Servers/swagger.yaml

```
@@ -2717,246 +2707,1370 @@ paths:
+      properties:
+        id:
+          type: number
+          example: 1
+        project_title:
+          type: string
+          example: "AI Compliance Checker"
+        owner:
+          type: string
+          example: "Alice"
+        users:
+          type: 
```

yamllint

[error] trailing spaces

(trailing-spaces)